docs/rst/interactive_application/getting_started/usage_example.rst

```diff
@@ -41,54 +41,121 @@ Spy configuration
 
 Additionally, it is possible to change the default configuration parameters by means of a YAML configuration file.
 
+.. TODO refer to Configuration
+
 .. note::
     Please refer to Configuration for more information on how to configure a |espy|.
 
 Spy execution
 =============
 
-Launching a |espy| instance is as easy as executing the following command:
+Source the following file to setup the |espy| environment:
+
+.. code-block:: bash
+
+    source install/setup.bash
+
+Launch an |espy| instance as executing the following command:
 
 .. code-block:: bash
 
-    fastdds_spy
+    fastddsspy
 
 Try out all the commands DDS Spy has to offer:
 
 * ``participants``
 
-.. figure:: /rst/figures/example_usage/spy_participants.png
-    :align: center
-    :scale: 75 %
+.. code-block:: bash
+
+    Insert a command for Fast DDS Spy:
+    >> participants
+    - name: TypeIntrospectionExample_Participant_Publisher
```

Comment:
If ShapesDemo is the one chosen to do the example, it would be better to have code snippets of such example.
Nevertheless, take a look to this PR https://github.com/eProsima/Fast-DDS-spy/pull/16
There are already ShapesDemo snippets that can be reused.